app/src/main/java/com/nervousfish/nervousfish/activities/FirstUseActivity.java

```diff
@@ -0,0 +1,54 @@
+package com.nervousfish.nervousfish.activities;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+
+import com.nervousfish.nervousfish.ConstantKeywords;
+import com.nervousfish.nervousfish.R;
+import com.nervousfish.nervousfish.service_locator.IServiceLocator;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The main activity class that shows a list of all people with their public keys
+ */
+public final class FirstUseActivity extends AppCompatActivity {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger("FirstUseActivity");
+
+    private IServiceLocator serviceLocator;
+
+    /**
+     * Creates the new activity, should only be called by Android
+     *
+     * @param savedInstanceState Don't touch this
+     */
+    @Override
+    protected void onCreate(final Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        this.setContentView(R.layout.activity_first_use);
+        final Intent intent = getIntent();
+        this.serviceLocator = (IServiceLocator) intent.getSerializableExtra(ConstantKeywords.SERVICE_LOCATOR);
+
+        if (getSupportActionBar() != null) {
+            getSupportActionBar().setDisplayShowTitleEnabled(false);
```

Comment:
It seems you have removed this in other classes, can you remove it here as well? 😇